spec/gemstash/storage_spec.rb

```diff
@@ -15,13 +15,6 @@
     expect(Gemstash::Storage.new(@folder)).not_to be_nil
   end
 
-  it "builds the path if it does not exists" do
```

Comment:
Why we don't need this specs?

Reply:
I re-add the test and change a bit the test itself since I moved the folder creation.